recipes/serapis/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "serapis" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Serapieum-of-alex/serapis/archive/{{ version }}.tar.gz
+  sha256: 46c2120d048c780d6fc1c1ce0ca4c991d849869ca537dc44b663a625f410e1b2
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+
+  host:
+    - python >=3.11
+    - pip >=23.3.1
+  run:
+    - python >=3.11
+    - numpy >=1.25.2
+    - pip >=23.3.1
+    - pyramids >=0.5.4
+    - geopandas >=0.14.1
+    - pandas >=2.1.0
+    - matplotlib-base >=3.7.2
+    - cleopatra >=0.4.0
+    - serapeum_utils >=0.2.0
+    - statista >=0.5.0
+    - statsmodels >=0.14.1
+    - loguru >=0.7.2
+    - pathlib >=1.0.1
+
+    
+test:
+  imports:
+    - serapis
+
+about:
+  home: https://github.com/Serapieum-of-alex/Serapis
+  license: GPL-3.0-only
+  license_file: LICENSE.md
+  license_family: GPL-3
+  license_file: LICENSE.md
+  summary: Flood modelling package
+  description: |
+    This module builds hydrodynamic model.
+  dev_url: https://github.com/Serapieum-of-alex/Serapis
+  doc_url: https://serapis.readthedocs.io/
```

Comment:
This URL is a 404 for me. Can you check if that is correct?

Reply:
removed incorrect link